tests/StringHelperTest.php

```diff
@@ -116,10 +116,19 @@ public function testTruncateWords(): void
      */
     public function testStartsWith(bool $result, string $string, ?string $with): void
     {
-        // case sensitive version check
         $this->assertSame($result, StringHelper::startsWith($string, $with));
-        // case insensitive version check
-        $this->assertSame($result, StringHelper::startsWith($string, $with, false));
+    }
+
+    /**
+     * @dataProvider providerStartsWith
+     * @param bool $result
+     * @param string $string
+     * @param string|null $with
+     */
+    public function testStartsWithCaseInsensitive(bool $result, string $string, ?string $with): void
+    {
+        $this->markTestIncomplete('Need to change data provider');
```

Comment:
Does that mean that in order to run the test I should do local modifications? That does not look good.

Reply:
My omission, will change data provider

Reply:
There was no need to change data provider. I've added an incorrect duplicate of another test.